### PR TITLE
[6.1] Update link

### LINKED ISSUE #46823: [6.1] Links to old docs site
PR #46786 updates the majority of the links to the old docs site to similar docs on the replacement guide site. There are however some that the docs team (@HermanPeeren et al) are still working on and they are listed below

Links that need to be updated/created and NOT changed in this PR

| File Path | Link Title | Link URL |
|-----------|------------|----------|
| .\administrator\components\com_categories\tmpl\categories\emptystate.php | Category | https://docs.joomla.org/Special:MyLanguage/Category |
| .\administrator\components\com_checkin\tmpl\checkin\emptystate.php | Help5.x:Maintenance: Global Check-in | https://docs.joomla.org/Special:MyLanguage/Help5.x:Maintenance:_Global_Check-in |
| .\administrator\components\com_finder\tmpl\maps\emptystate.php | Help5.x:Smart Search: Content Maps | https://docs.joomla.org/Special:MyLanguage/Help5.x:Smart_Search:_Content_Maps |
| .\administrator\components\com_finder\tmpl\searches\emptystate.php | Help5.x:Smart Search: Search Term Analysis | https://docs.joomla.org/Special:MyLanguage/Help5.x:Smart_Search:_Search_Term_Analysis |
| .\administrator\components\com_guidedtours\tmpl\steps\emptystate.php | Help5.x:Guided Tours: Steps | https://docs.joomla.org/Special:MyLanguage/Help5.x:Guided_Tours:_Steps |
| .\administrator\components\com_guidedtours\tmpl\tours\emptystate.php | Help5.x:Guided Tours | https://docs.joomla.org/Special:MyLanguage/Help5.x:Guided_Tours |
| .\administrator\components\com_installer\tmpl\discover\emptystate.php | Help5.x:Extensions: Discover | https://docs.joomla.org/Special:MyLanguage/Help5.x:Extensions:_Discover |
| .\administrator\components\com_installer\tmpl\update\emptystate.php | Help5.x:Extensions: Update | https://docs.joomla.org/Special:MyLanguage/Help5.x:Extensions:_Update |
| .\administrator\components\com_installer\tmpl\warnings\emptystate.php | Help5.x:Information: Warnings | https://docs.joomla.org/Special:MyLanguage/Help5.x:Information:_Warnings |
| .\administrator\components\com_joomlaupdate\tmpl\update\default.php | J4.x:Joomla Update Problems | https://docs.joomla.org/Special:MyLanguage/J4.x:Joomla_Update_Problems |
| .\administrator\components\com_messages\tmpl\messages\emptystate.php | Help5.x:Private Messages | https://docs.joomla.org/Special:MyLanguage/Help5.x:Private_Messages |
| .\administrator\components\com_privacy\tmpl\consents\emptystate.php | Help5.x:Privacy: Consents | https://docs.joomla.org/Special:MyLanguage/Help5.x:Privacy:_Consents |
| .\administrator\components\com_privacy\tmpl\requests\emptystate.php | Help5.x:Privacy: Information Requests | https://docs.joomla.org/Special:MyLanguage/Help5.x:Privacy:_Information_Requests |
| .\administrator\components\com_redirect\tmpl\links\emptystate.php | Help5.x:Redirects: Links | https://docs.joomla.org/Special:MyLanguage/Help5.x:Redirects:_Links |
| .\administrator\components\com_scheduler\tmpl\tasks\empty_state.php | J4.x:Task Scheduler | https://docs.joomla.org/Special:MyLanguage/J4.x:Task_Scheduler |
| .\administrator\components\com_users\tmpl\notes\emptystate.php | Help5.x:User Notes | https://docs.joomla.org/Special:MyLanguage/Help5.x:User_Notes |
|~.\administrator\language\en-GB\com_cpanel.ini~ | ~Portal:Beginners~ | ~https://docs.joomla.org/Special:MyLanguage/Portal:Beginners\~ |
| .\administrator\language\en-GB\com_installer.ini | Help4.x:Information: Warnings | https://docs.joomla.org/Special:MyLanguage/Help4.x:Information:_Warnings\ |
|~.\administrator\language\en-GB\com_joomlaupdate.ini~ | ~Pre-Update Check~ | ~https://docs.joomla.org/Special:MyLanguage/Pre-Update_Check~ |
| .\build\layouts\github.php | How to determine a package checksum | https://docs.joomla.org/How_to_determine_a_package_checksum |
| .\build\layouts\github.php | How to determine a package checksum | https://docs.joomla.org/How_to_determine_a_package_checksum |
| ~.\cli\joomla.php~ | ~J5.x:Setting Up Your Local Environment~ | ~https://docs.joomla.org/Special:MyLanguage/J5.x:Setting_Up_Your_Local_Environment~ |
| .\components\com_ajax\ajax.php | Supporting plugins in your component | https://docs.joomla.org/Special:MyLanguage/Supporting_plugins_in_your_component |
| ~.\installation\includes\cli.php~ | ~J5.x:Setting Up Your Local Environment~ | ~https://docs.joomla.org/Special:MyLanguage/J5.x:Setting_Up_Your_Local_Environment~ |
| ~.\installation\language\en-GB\joomla.ini~ | see #46794 ~Joomla and MySQL 8: Workaround~| ~https://docs.joomla.org/Special:MyLanguage/Joomla_and_MySQL_8#Workaround_to_get_Joomla_working_with_MySQL_8\~|
| .\installation\src\Helper\DatabaseHelper.php | J3.x:Secured procedure for installing Joomla with a remote database | https://docs.joomla.org/Special:MyLanguage/J3.x:Secured_procedure_for_installing_Joomla_with_a_remote_database |
| .\libraries\src\Layout\BaseLayout.php | Sharing layouts across views or extensions with JLayout | https://docs.joomla.org/Special:MyLanguage/Sharing_layouts_across_views_or_extensions_with_JLayout |
| .\libraries\src\Layout\FileLayout.php | Sharing layouts across views or extensions with JLayout | https://docs.joomla.org/Special:MyLanguage/Sharing_layouts_across_views_or_extensions_with_JLayout |
| .\libraries\src\Layout\LayoutHelper.php | Sharing layouts across views or extensions with JLayout | https://docs.joomla.org/Special:MyLanguage/Sharing_layouts_across_views_or_extensions_with_JLayout |
| .\libraries\src\Layout\LayoutInterface.php | Sharing layouts across views or extensions with JLayout | https://docs.joomla.org/Special:MyLanguage/Sharing_layouts_across_views_or_extensions_with_JLayout |
|~.\plugins\quickicon\eos\src\Extension\Eos.php~ | ~Pre-Update Check~ | ~https://docs.joomla.org/Special:MyLanguage/Pre-Update_Check~|

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request **partialy** resolves #46823 .

- [x] I read the [Generative AI policy](https://developer.joomla.org/generative-ai-policy.html) and my contribution is either not created with the help of AI or is compatible with the policy and GNU/GPL 2 or later.

### Summary of Changes
**Continues** the work to update the links from the old docs site to the new guide site. I have now marked these as resolved on #46823
Changes the link from https://docs.joomla.org/Special:MyLanguage/J5.x:Setting_Up_Your_Local_Environment to https://guide.joomla.org/user-manual/hosting/local-environment-setup

### Testing Instructions
code review

### Actual result BEFORE applying this Pull Request
old link to docs site

### Expected result AFTER applying this Pull Request
new link to same content on guide site

### Link to documentations
Please select:
- [ ] Documentation link for guide.joomla.org: <link>
- [x] No documentation changes for guide.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Review by tecpromotion — APPROVED** [review]
Thanks @brianteeman
